src/main/java/com/pingcap/tikv/expression/TiColumnRef.java

```diff
@@ -70,7 +70,13 @@ public Expr toProto() {
     Expr.Builder builder = Expr.newBuilder();
     builder.setTp(ExprType.ColumnRef);
     CodecDataOutput cdo = new CodecDataOutput();
-    IntegerType.writeLong(cdo, columnInfo.getId());
+    // After switching to DAG request mode, expression value
```

Comment:
What does this mean?

Reply:
A DAG request consists of several executors and are organized by specific order. The first executor type should be `TypeTableScan` or `TypeIndexScan`, in this very first executor, we specify the columns that will be used for calculation and store them as a `List`, for the following executors, if they need to reference any column set in the first executor, they need to set this `ColumnRef`'s value to the index of the columns `List` specified in the first executor.